Week01/index.html

```diff
@@ -4,7 +4,58 @@
         <meta charset="UTF-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Document</title>
+
+        <link rel="stylesheet" href="./style.css" />
+        <title>홍맹구의 To Do List!</title>
     </head>
-    <body></body>
+    <body>
+        <!-- HEADER -->
+        <header>
+            <h1>Todo List</h1>
+        </header>
+
+        <!-- TOP NAV -->
+        <nav>
+            <div class="btn-group">
+                <button>오늘만 보기</button>
+                <button>내일만 보기</button>
+                <button>함께 보기</button>
+            </div>
+        </nav>
+
+        <main>
+            <!-- LEFT SECTION -->
+            <section id="left-section">
+                <ul>
+                    오늘할일
+                    <li>1</li>
+                    <li>2</li>
+                </ul>
+
+                <div class="add">
+                    <input type="text" />
+                    <button>+</button>
+                </div>
```

Comment:
여기서도 살짝 시멘틱구조로 바꾸면 위에 li들이 input과 button에 대한 사용자의 입력 및 전송으로 목록들이 추가데자나! 그래서 [form태그](https://www.nextree.co.kr/p8428/)가 적합할듯해!! 

```suggestion
 <form  class="add-form" action="http..." accept-charset="utf-8" name="todos_info" method="post" >
       <input type="text" />
        <button>+</button>
</form>
``` 
폼속성이 뭐가 있는지도 공부해봤으면 좋겟당 ㅎㅎ